src/components/card/CardGroup/CardGroup.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import { CardGroup } from './CardGroup'
+
+describe('CardGroup component', () => {
+  it('renders without errors', () => {
+    const { queryByTestId } = render(<CardGroup />)
+    expect(queryByTestId('CardGroup')).toBeInTheDocument()
+  })
+
+  it('renders its children', () => {
+    const { queryByText } = render(<CardGroup>My Content</CardGroup>)
```

Comment:
non-blocking, but this may result in a warning since a `<ul>` should only have `<li>` as children